src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -6643,6 +6643,70 @@ .locals init (CustomHandler V_0)
 ");
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/79888")]
+        public void PassAsRefWithoutKeyword_05()
+        {
+            var code = """
+                BadFunc(
```

Comment:
>BadFunc [](http://example.com/codeflow?start=16&length=7)

Consider also testing without changing arguments order. I assume ```BuildStoresToTemps``` won't be used then.

Reply:
Added.